### PR TITLE
Added story for `dnn-checkbox`

### LINKED ISSUE #505: Implement story for `dnn-checkbox`
Implement story for `dnn-checkbox`

### PR DESCRIPTION
**Pull request by david-poindexter**
Resolves #505 

image

### CODE REVIEW

File: src/components/dnn-checkbox/dnn-checkbox.stories.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { html } from "lit-html";
+import { ifDefined } from 'lit-html/directives/if-defined';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
```
Comment (valadas) [inline src/components/dnn-checkbox/dnn-checkbox.stories.ts:3]:
```suggestion
```

This is unused the the linter makes the build fail

**Review by valadas — CHANGES_REQUESTED** [review]

File: src/components/dnn-checkbox/dnn-checkbox.stories.ts (hunk 1 above)
Comment (david-poindexter) [reply src/components/dnn-checkbox/dnn-checkbox.stories.ts:3]: I thought I removed that - guess I didn't.  Good catch!

Comment (david-poindexter) [reply src/components/dnn-checkbox/dnn-checkbox.stories.ts]: I just pushed a fix commit.

**Review by valadas — APPROVED** [review]
Thank you sir!

**Pull request merged**